nautobot/core/templates/generic/object_list.html

```diff
@@ -17,6 +17,38 @@
 </style>
 {% endblock extra_styles %}
 
+{% block header %}
+    <div class="row noprint">
+        {% with list_url=content_type.model_class|validated_viewname:"list" %}
+        <div class="col-sm-8 col-md-9">
+            <ol class="breadcrumb">
+            {% block breadcrumbs %}
+                {% if list_url %}
+                <li><a href="{% url list_url %}">
+                    {{ content_type.model_class|meta:'verbose_name_plural'|bettertitle }}
+                </a></li>
+                {% endif %}
+                {% block extra_breadcrumbs %}{% endblock extra_breadcrumbs %}
+            {% endblock breadcrumbs %}
+            </ol>
+        </div>
+        <div class="col-sm-4 col-md-3">
+            <form action="#" method="get" id="search-form">
+                <div class="input-group">
+                    {{ search_form.q }}
```

Comment:
Do we really need a `search_form` at all, given we're really only just using this widget? Seems like it might be simpler to hard-code it into the template, similar to what we're already doing in object_retrieve.html, a la:

```
<input type="text" name="q" class="form-control" placeholder="Search {{ content_type.model_class|meta:'verbose_name_plural' }}">
```

Reply:
With this method, I will have to manually handle the input value for "q" if the query string contains it.

Reply:
Fair point. Aren't we already doing something like that for all the other query parameters in the parent branch though?

Reply:
No